schemas/oval/5.11.3/oval-definitions-schema.xsd

```diff
@@ -1553,13 +1553,8 @@
             <xsd:extension base="xsd:anySimpleType">
                 <xsd:attribute name="name" use="required">
                     <xsd:annotation>
-                        <xsd:documentation>A string restricted to disallow upper case characters.</xsd:documentation>
```

Comment:
I'm afraid that changing the core part of schema will be difficult.

Reply:
I'm afraid that it is the only sane way. Anyhow, until OVAL decides something, I think it would be the best option to stick with the simplest implementation.